src/components/Compare/Compare.style.tsx

```diff
@@ -209,14 +224,19 @@ export const TableHeadContainer = styled(TableHead)<{ isModal?: Boolean }>`
     color: ${({ isModal }) => isModal && 'white'};
     border-bottom: ${({ isModal }) =>
       !isModal && `2px solid ${COLORS.LIGHTGRAY}`};
-
-    &:nth-child(2) {
-      border-left: ${({ isModal }) =>
-        !isModal && `2px solid ${COLORS.LIGHTGRAY}`};
-    }
   }
 `;
 
+export const Population = styled.span`
+  font-family: Source Code Pro;
+  font-size: 0.875rem;
+`;
+
+export const CountySuffix = styled.div`
```

Comment:
nit: maybe you want a span here?

Reply:
spans within a MetricCell have specific styling (metric values) so it was less involved to just make it a div